src/TinyGsmClient.h

```diff
@@ -35,6 +35,7 @@ typedef TinyGsmSim7000::GsmClientSim7000 TinyGsmClient;
 #include "TinyGsmClientSIM7020.h"
 typedef TinyGsmSim7020                   TinyGsm;
 typedef TinyGsmSim7020::GsmClientSim7020 TinyGsmClient;
+typedef TinyGsmSim7020::GsmClientSecureSim7020 TinyGsmClientSecure;
```

Comment:
A test was failing because 7020 is declared (macro) to support SSL, but the typedef was missing